scripts/guests.js

```diff
@@ -10,4 +10,21 @@
                 Add park area id as state
 
         Return HTML String
-*/
\ No newline at end of file
+*/
+
+import { getGuestsList } from "./database.js"
+
+
+export const guestsList = () => {
+    const guests = getGuestsList()
+    let htmlString = `<div class="guests">`
+
+    for (const guest of guests) {
+        htmlString += `<div class="guestList">
```

Comment:
I believe that the naming convention for HTML uses all lower case, with dashes between words.

I am a little confused on your chose name for this class. Being inside of a forof loop, each individual guest would have the class `guestList`. At least to me, it feels like that would make sense as the class name for the Element containing all of the guests, rather than a single guest. Maybe something like `guest-item` would suffice?